File: Python/parking2.py
'''
 *
 *	Problem: "Parking"
 *
'''

cases = int(input())

totalArea = 0
for x in range(cases):
	numStores = int(input())
	
	stores = input().split()
	maxPos = int(stores[0])
	minPos = int(stores[0])

	for y in range(numStores):
		currStore = int(stores[y])
		if (currStore > maxPos):
			maxPos = currStore
		if (currStore < minPos):
			minPos = currStore
	
	result = (maxPos - minPos) * 2
	print(result)

File: Java/parking2.java
/*
 *	Problem: "Parking"
 */

import java.util.Scanner;

public class parking2 {

	public static void mySolution(Scanner sc, int cases) {
		for (int i = 0; i < cases; i++) {
			int numStores = sc.nextInt();
			
			int currentStore = sc.nextInt();
			int maxPos = currentStore;
			int minPos = currentStore;
			for (int j = 0; j < numStores - 1; j++) {
				currentStore = sc.nextInt();
				if ( currentStore > maxPos)
					maxPos = currentStore;
				if ( currentStore < minPos)
					minPos = currentStore;
			}
			
			int result = (maxPos - minPos) * 2;
			System.out.println(result);
		}
	}

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int cases = sc.nextInt();
		mySolution(sc, cases);
	}
	
}